src/elements/common/content-answers/ContentAnswersModal.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useState, useCallback } from 'react';
+import { useIntl } from 'react-intl';
+import getProp from 'lodash/get';
+import { AxiosResponse } from 'axios';
+
+import {
+    AgentType,
+    ANSWER_ERROR,
+    IntelligenceModal,
+    QuestionType,
+    SuggestedQuestionType,
+} from '@box/box-ai-content-answers';
+
+import { DOCUMENT_SUGGESTED_QUESTIONS, SPREADSHEET_FILE_EXTENSIONS } from './constants';
+import withCurrentUser from '../current-user';
+
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { BoxItem, User } from '../../../common/types/core';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { withAPIContext } from '../api-context';
+// @ts-ignore: no ts definition
+import APIFactory from '../../../api';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../common/types/api';
+
+import messages from './messages';
+
+export interface ExternalProps {
+    isCitationsEnabled?: boolean;
+    isMarkdownEnabled?: boolean;
+    isResetChatEnabled?: boolean;
+    onAsk?: () => void;
+    onClearConversation?: () => void;
+    onRequestClose?: () => void;
+    suggestedQuestions?: SuggestedQuestionType[];
+}
+
+export interface ContentAnswersModalProps extends ExternalProps {
+    api: APIFactory;
+    currentUser?: User;
+    file: BoxItem;
+    isOpen: boolean;
+}
+
+const ContentAnswersModal = ({
+    api,
+    currentUser,
+    file,
+    isOpen,
+    onAsk,
+    onClearConversation,
+    onRequestClose,
+    suggestedQuestions,
+    isCitationsEnabled = true,
+    isMarkdownEnabled = true,
+    isResetChatEnabled = true,
+}: ContentAnswersModalProps) => {
+    const { formatMessage } = useIntl();
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+    const [questions, setQuestions] = useState<QuestionType[]>([]);
+    let localizedQuestions: SuggestedQuestionType[] = [];
+
+    if (!suggestedQuestions) {
```

Comment:
fyi an empty array for `suggestedQuestions` is truthy so this only enters the `if` statement for `undefined`. is that okay?

my guess is that it could render empty space but that seems out of scope for this component and more of something that should be handled by the shared-feature